hacking/aws_config/testing_policies/redshift-policy.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowRedshiftManagment",
```

Comment:
Can you please add this policy to `hacking/aws_config/testing_policies/database-policy.json`?  The AWS default policy quota is fairly low so we try to keep these condensed.